src/includes/getting-started-verify/javascript.angular.mdx

```diff
@@ -0,0 +1,3 @@
+```javascript
```

Comment:
this will automatically use the javascript file, you dont have to define it for angular

Reply:
Thanks. Will remove.